src/main/webapp/my-profile.html

```diff
@@ -55,13 +55,22 @@ <h2>Edit Profile</h2>
               <input type="text" id="name" name="name" size="40" required></input>
               <label for="Department">I am currently a student in the department of</label>
```

Comment:
```suggestion
              <label for="Department">I am a student in the department of</label>
```

Just to match the one in the form